qparam_test.go

```diff
@@ -247,4 +249,36 @@ func TestReader_Read(t *testing.T) {
 		assert.NoError(t, err)
 		assert.EqualValues(t, expected, target)
 	})
+
+	t.Run("Format", func(t *testing.T) {
```

Comment:
Please put those sub-tests in a separate test function. For example:
```go
func TestMultiError_Format(t *testing.T) { ... }
```
Also: please use descriptive test names for sub tests:
```go
t.Run("format with %s", func(t *testing.T) { ... }
```